nextflow.config

```diff
@@ -24,6 +24,7 @@ params {
     analysis_type              = 'wgs'
     annotate_snv_switch        = true
     annotate_sv_switch         = false
+    annotate_mt_switch         = true
```

Comment:
I would rename this parameter. We're not only annotating MT variants, we are also aligning and calling. Perhaps `dedicated_mt_workflow`?

Reply:
good idea 👍🏻